react/PageLayoutContainer.tsx

```diff
@@ -20,13 +20,13 @@ const PageLayoutContainer: FC = ({ children }) => {
       <div className="flex flex-row-l flex-column vh-100-l">
         <EnhancedSideBarContentProvider>
           <div
-            className="w-25-l vh-100-l overflow-y-scroll"
-            style={{ maxWidth: '280px', minWidth: '200px' }}>
+            className="w-25-l vh-100-l overflow-y-auto"
+            style={{ maxWidth: 280, minWidth: 200 }}>
```

Comment:
It is preferred to use `'280px'` instead of just the number. I think React will deprecate this.

Reply:
Awesome! Did not know about that!